### DISCUSSION TITLE
Upload releases

### DISCUSSION THREAD

**Discussion by kleo** [linux-3ds/arm9linuxfw#10] [General]
Hello! Github actions creates a build but does not automatically puts it on Releases. You have to manually download it and upload as Release. I can upload it for you guys but I don't have enough experience to provide info on the changes you have made on the repo as changelog.

**Comment by Wolfvak** [comment]
I'm sure there must be some way to link to the latest artifact build from master.

**Reply by Wolfvak** [reply]
@kleo I just pushed a commit to a new branch and it made a new release, can you fix it so that it'll only make those when pushing to master (or merging to it)?

**Reply by kleo** [reply]
Done on https://github.com/linux-3ds/arm9linuxfw/pull/12, https://github.com/linux-3ds/firm_linux_loader/pull/22 and https://github.com/linux-3ds/linux/pull/46
